### DISCUSSION TITLE
Loading Animation

### DISCUSSION THREAD

**Discussion by boredjo** [evidence-dev/evidence#2692] [Q&A]
How can I add a loading animation, so that it is obvious that the evidence page is still loading and not just unresponsive?

**Comment by hughess** [comment]
Hi @boredjo what's the situation where you need the loading animation?

**Comment by boredjo** [comment]
When the source datasets are huge, and the server, connection, etc. is slow, then It would be nice to have a loading animation to cover the dashboard so that no empty dataset warnings appear. I know there is a loading footnote, but that is not reactive enough. Ideally, I would like a progress bar, but a simple rotating circle is enough to distinguish it from a frozen site (or at least make people want to wait a little longer before trying to refresh or move on)
